From: Tomaz Zaman <tomaz@mono.si>
Date: Thu, 21 Aug 2025 12:00:00 +0000
Subject: [PATCH] arm64: dts: freescale: build the Mono Gateway DK dtb

Add the Mono Gateway DK board to the freescale dtb build list. The board
.dts lives in the layerscape target's files/ overlay; this makes the
kernel compile mono-gateway-dk.dtb so the image build can pick it up.

Signed-off-by: Tomaz Zaman <tomaz@mono.si>
---
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -21,6 +21,7 @@
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1046a-qds.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1046a-rdb.dtb
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1046a-tqmls1046a-mbls10xxa.dtb
+dtb-$(CONFIG_ARCH_LAYERSCAPE) += mono-gateway-dk.dtb
 DTC_FLAGS_fsl-ls1088a-qds := -Wno-interrupt_map
 dtb-$(CONFIG_ARCH_LAYERSCAPE) += fsl-ls1088a-qds.dtb
 DTC_FLAGS_fsl-ls1088a-rdb := -Wno-interrupt_map
